src/components/CloudHeader/CloudHeader.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+const CloudHeader = props => {
+  const { children, className, ...other } = props;
+
+  const cloudHeaderClasses = classNames('cloud-header', className);
```

Comment:
This pattern shows up so often, I'm curious if we should just have a wrapper around this that provides this behavior for us. Like:

```
<Cx defaultClassName="cloud-header" element="nav" {...other} />
```